x-pack/plugins/security_solution/cypress/integration/detection_rules/export_rule.spec.ts

```diff
@@ -86,4 +88,51 @@ describe('Export rules', () => {
       `Successfully exported ${expectedNumberCustomRulesToBeExported} of ${totalNumberOfRules} rules. Prebuilt rules were excluded from the resulting file.`
     );
   });
+
+  context('rules with exceptions', () => {
+    before(() => {
+      deleteExceptionList(getExceptionList().list_id, getExceptionList().namespace_type);
```

Comment:
Let's extract duplicated `getExceptionList()` calls to a variable.

Reply:
done